x-pack/plugins/endpoint/public/applications/endpoint/store/alerts/mock_alert_result_list.ts

```diff
@@ -26,19 +26,24 @@ export const mockAlertResultList: (options?: {
   const alerts = [];
   for (let index = 0; index < actualCountToReturn; index++) {
     alerts.push({
-      '@timestamp': new Date(1542341895000).toString(),
+      '@timestamp': new Date(1542341895000),
+      id: 'xDUYMHABAJk0XnHd8rrd',
```

Comment:
Exposed the alert document's `_id` field here as `id`. To be used for retrieval via the details API.